### ISSUE TITLE
Tooltip window contents not reachable using the keyboard

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon#5365]
## Environment

> MacOS

> Firefox

> Keyboard

This happens with the latest version of what is exposed here: https://react.carbondesignsystem.com

## Detailed description

> When an interactive tooltip (a tooltip that contains buttons or links) is navigated to and opened using a keyboard, the contents (a link or button) of the tooltip window is not reachable using the keyboard. The only action possible using keyboard is closing the tooltip window again.

**Note:** It seems, this is a carbon-component-react issue since with carbon-design-system, the contents of the tooltip window seem reachable (from what I can tell from their page carbondesignsystem.com and its respective sample).

> What should happen: The contents of a tooltip window IS reachable using a keyboard

> What happens: the only thing that happens is that the tooltip window can be closed.

## Steps to reproduce the issue

1. open https://react.carbondesignsystem.com/?path=/story/tooltip--default-bottom
2. Navigate to the tooltip icon
3. press space or enter to open the tooltip window
4. press TAB again to select the link or button within the tooltip window 

## Additional information

![image](https://user-images.githubusercontent.com/51151716/74655010-de4b4400-518b-11ea-8b58-b53caac03360.png)

**Comment by emyarod** [comment]
related https://github.com/carbon-design-system/carbon/issues/1264 https://github.com/carbon-design-system/carbon/issues/4414
